pkg/splunk/enterprise/util.go

```diff
@@ -234,16 +233,6 @@ func ApplySplunkConfig(ctx context.Context, client splcommon.ControllerClient, c
 	return namespaceScopedSecret, nil
 }
 
-// getIndexerExtraEnv returns extra environment variables used by indexer clusters
```

Comment:
I take it these are removed due to not being used.  So this functionality has moved or is not longer needed?

Reply:
Currently we don't need the function/functionality. Per the [PR](https://github.com/splunk/splunk-operator/pull/244), we can reintroduce it if required.